docs/blog/2020-11-05-zmk-sotf-3.md

```diff
@@ -0,0 +1,156 @@
+---
+title: "ZMK State Of The Firmware #3"
+author: Pete Johanson
+author_title: Project Creator
+author_url: https://gitlab.com/petejohanson
+author_image_url: https://www.gravatar.com/avatar/2001ceff7e9dc753cf96fcb2e6f41110
+tags: [SOTF, keyboards, firmware, oss, ble]
+---
+
+Welcome to the third ZMK "State Of The Firmware" (SOTF)!
+
+This update will cover all the major activity since [SOTF #2](/blog/2020/09/21/zmk-sotf-2). This edition comes a bit later than planned, but the amount of features and changes will hopefully make it worth it!
+
+## Recent Activity
+
+Here's a summary of the various major changes since last time, broken down by theme:
+
+### Keymaps/Behaviors
+
+Tons of activity related to keymaps, so we'll go into more detail this time.
+
+#### Codes Overhaul
+
+Contributor [innovaker] _completely_ overhauled the set of available codes you can use in keymaps, and simultaneously has created
+_beautiful_ [documentation](/docs/codes/) to help users see the available codes, and also understand if they are supported on their particular operating system.
+
+This also laid the foundation for the other keymap related changes that are now available:
+
+#### Modified (Shifted) Codes
+
+Contributor [okke-formsma] added the ability to send additional modifiers along with your base code, e.g.:
+
+```
+&kp LC(C)
+```
+
+Which will send `Control + c` when pressed. This feature is often used on smaller keyboard to achieve "shifted keycodes", e.g. `LS(N1)` to send a `!`.
+To make this easier, in addition to all the normal codes, we know have defines for common shifted codes, e.g. `EXCL` for `!`, `AT` for `@`, etc.
+
+To learn more checkout the [Modifiers](/docs/codes/modifiers) documentation.
+
+#### Simplified Key Press Behavior
+
+In previous versions of ZMK, users needed to be careful to select between the `&kp` and `&cp` behavior in their keymaps, depending on
+if the particular keycode they wanted to send was in the "HID consumer page" or not. Forcing user to understand the difference, and get
```

Comment:
```suggestion
if the particular keycode they wanted to send was in the "HID consumer page" or not. Forcing user to understand the difference and get
```
English is a dumb language and I am by no means an expert, but I think the comma here is unnecessary. The first time I read this, the comma got me to try to parse "and get this right was awkward and error prone" as a standalone phrase, and I was confused until I re-read it a couple times.

Reply:
Thanks. Fixed.